WordPress/Classes/ViewRelated/Blog/SiteSettingsViewController.m

```diff
@@ -97,6 +108,11 @@ - (void)viewDidLoad
     [super viewDidLoad];
 
     self.navigationItem.title = NSLocalizedString(@"Settings", @"");
+    if ([self.blog isHostedAtWPcom]) {
```

Comment:
More generally, `isHostedAtWPcom` should be used very sparingly, and rely on `-[Blog supports:]` instead. That way, when things start to be supported in Jetpack there's only one method to change.